Snip/Resources/Strings.txt

```diff
@@ -41,6 +41,10 @@ mpsytIsNotRunning=mps-youtube is not currently running
 ; selected media file or when the user stops/pauses a playing track.
 NoTrackPlaying=No Track Playing
 
+; This text is saved to the Snip.txt file when mpsyt apears to have no title 
```

Comment:
This is really unnecessary. The instructions should be clear in README.md.
Just display NoTrackPlaying instead.